src/kv.ts

```diff
@@ -39,22 +40,25 @@ export function parseKVString(input: string): KVPair {
 
   const result: KVPair = {};
 
-  // This regular expression uses a lookahead to split on commas which are not
-  // preceeded by an escape character (slash).
-  const pairs = input.split(/(?<!\\),/gi);
+  // This regular expression uses a lookahead to split on commas and newlines
+  // which are not preceeded by an escape character (slash).
+  const pairs = input.split(/(?<!\\)[,\n]/gi);
   for (let i = 0; i < pairs.length; i++) {
     const pair = pairs[i];
     const firstEqual = pair.indexOf('=');
     if (!firstEqual || firstEqual === -1) {
       throw new SyntaxError(`Failed to parse KEY=VALUE pair "${pair}": missing "="`);
     }
 
-    // Trim any key whitespace and un-escape any escaped commas.
-    const k = pair.slice(0, firstEqual).trim().replace(/\\,/gi, ',');
+    // Trim any key whitespace and un-escape any escaped commas and newlines.
+    const k = pair
+      .slice(0, firstEqual)
+      .trim()
+      .replace(/\\([,\n])/gi, '$1');
```

Comment:
nit: should `/\\([,\n])/gi` be a variable? Or do we think this will change much in the future?